docs-new/docs/graphql/core/deployment/production-checklist.mdx

```diff
@@ -41,31 +40,29 @@ HASURA_GRAPHQL_ADMIN_SECRET=averylongpasswordstring
 graphql-engine --database-url=<database-url> serve --admin-secret=averylongpasswordstring
 ```
 
-More details can be found at `securing_graphql_endpoint`.
+More details can be found at [Securing the GraphQL endpoint](/graphql/core/deployment/securing-graphql-endpoint.mdx/).
 
 ## Verify permissions
 
-<div class="contents" backlinks="none" depth="1" local="">
-
-</div>
+- [Review the summary](/graphql/core/deployment/production-checklist.mdx#review-the-summary)
+- [Limit number of rows returned](/graphql/core/deployment/production-checklist.mdx#limit-number-of-rows-returned)
 
-### Review the summary
+### Review the summary {#review-the-summary}
```

Comment:
These links are actually table of contents (TOC) for the sub section.

Please use - `TOCInline` component which will auto generate the subsection link list.

The below should work for this doc.
```jsx
<TOCInline toc={toc[2].children} />
```

Please refer the [Sub-Subitem section in guide](https://hge-docs-migration.netlify.app/migration-guide/table-of-contents#sub-subitem)

Also no need to attach custom id when its not added in original file.